Lib/fontTools/varLib/instancer/__init__.py

```diff
@@ -323,14 +323,11 @@ def limitTupleVariationAxisRange(var, axisTag, axisRange):
         return [var, newVar]
 
 
-def instantiateGvarGlyph(varfont, glyphname, axisLimits, optimize=True):
-    glyf = varfont["glyf"]
-    coordinates, ctrl = glyf.getCoordinatesAndControls(glyphname, varfont)
+def _instantiateGvarGlyph(glyphname, glyf, gvar, hMetrics, vMetrics, axisLimits, optimize=True):
```

Comment:
in this case I don't mind too much that the signature is different (api break), but I guess it would still be nice to export a non `_`-prefixed public `instantiateGvarGlyph` method. Imagine one wants to instantiate different glyphs and different positions for whatever reason